sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/Models/OSImageSku.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Workloads.Models
+{
+    /// <summary> OS image sku. </summary>
+    public readonly partial struct OSImageSku : IEquatable<OSImageSku>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="OSImageSku"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public OSImageSku(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string Eighteen04LTSValue = "18.04-LTS";
+        private const string Sixteen04LTSValue = "16.04-LTS";
+
+        /// <summary> 18.04-LTS. </summary>
+        public static OSImageSku Eighteen04LTS { get; } = new OSImageSku(Eighteen04LTSValue);
+        /// <summary> 16.04-LTS. </summary>
+        public static OSImageSku Sixteen04LTS { get; } = new OSImageSku(Sixteen04LTSValue);
```

Comment:
These names are not good, I would suggest:
```suggestion
        public static OSImageSku Ubuntu18_04Lts { get; } = new OSImageSku(Eighteen04LTSValue);
        /// <summary> 16.04-LTS. </summary>
        public static OSImageSku Ubuntu16_04Lts { get; } = new OSImageSku(Sixteen04LTSValue);
```